File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        i=len(s)-1
        count=0
        while(s[i]==" " and i>=0):
            i=i-1
        while(i>=0 and s[i]!=" "):
            count=count+1
            i=i-1
        return count

        
        

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        String ans=s.trim();
        int count=0;
        int n=ans.length()-1;
        while(n>=0 && ans.charAt(n)!=' '){
            n--;
            count=count+1;

        } 
        return count;
    }
}